src/data/polynomial/eval.lean

```diff
@@ -380,22 +380,19 @@ instance [decidable_eq R] : decidable (is_root p a) := by unfold is_root; apply_
 
 lemma is_root.eq_zero (h : is_root p x) : eval x p = 0 := h
 
-lemma coeff_zero_eq_eval_zero (p : R[X]) :
-  coeff p 0 = p.eval 0 :=
+lemma coeff_zero_eq_eval_zero (p : R[X]) : coeff p 0 = p.eval 0 :=
 calc coeff p 0 = coeff p 0 * 0 ^ 0 : by simp
 ... = p.eval 0 : eq.symm $
   finset.sum_eq_single _ (λ b _ hb, by simp [zero_pow (nat.pos_of_ne_zero hb)]) (by simp)
 
-lemma zero_is_root_of_coeff_zero_eq_zero {p : R[X]} (hp : p.coeff 0 = 0) :
-  is_root p 0 :=
+lemma zero_is_root_of_coeff_zero_eq_zero {p : R[X]} (hp : p.coeff 0 = 0) : is_root p 0 :=
 by rwa coeff_zero_eq_eval_zero at hp
 
 lemma is_root.dvd {R : Type*} [comm_semiring R] {p q : R[X]} {x : R}
   (h : p.is_root x) (hpq : p ∣ q) : q.is_root x :=
 by rwa [is_root, eval, eval₂_eq_zero_of_dvd_of_eval₂_eq_zero _ _ hpq]
 
-lemma not_is_root_C (r a : R) (hr : r ≠ 0) : ¬ is_root (C r) a :=
-by simpa using hr
+lemma not_is_root_C (r a : R) (hr : r ≠ 0) : ¬ is_root (C r) a := by simpa using hr
```

Comment:
I'm not going to block the PR over this, but I don't like changes like this for readability reasons: putting the type and proof on separate lines makes it easier to find each part.

Reply:
I'll stop doing this from now on, thanks